### DISCUSSION TITLE
Prevent reversing.

### DISCUSSION THREAD

**Discussion by Sabreklaw** [Courseplay/Courseplay_FS22#3028] [Ideas]
There are a lot of impliment combinations that can not be reversed.  Large seeders that have a seed/fertilizer wagon behind them for example.  Most of the time they just end up in a pile.  Can you make an option that will force Courseplay to only drive forward?  Even if it has to miss a spot to make the turn, that would be preferable to having to unpile the equipment.
![pileup](https://github.com/Courseplay/Courseplay_FS22/assets/96848273/67e46de4-b0c5-49be-8127-abfcf3349b9e)

**Comment by Tensuko** [comment]
Turn on field off -> then he will not do reverse.
